_docs/reference/release-roadmap.md

```diff
@@ -8,5 +8,28 @@ layout: docs
 type: markdown
 ---
 
-TBD
+## Istio v0.2
 
+Istio v0.2 will be our second release, and will work across multiple Kubernetes clusters and raw VMs, with a larger number of services. Istio v0.2 will support the following features:
+- TCP telemetry collection and policy enforcement.
+- Hybrid deployments (multiple clusters, VMs).
+- Transparent proxy injection in Kubernetes.
+- Standard extension points for addons and plugins.
+- Expanded Authorization with callouts to external services.
+- Request Tracing with Zipkin.
```

Comment:
+1 for moving this to the 0.1 feature set. We'll move to unary RPCs between Mixer and Envoy in a 0.1.x release (@geeknoid correct me if I'm wrong) which is the single blocker for tracing in Mixer. Tracing has already been implemented in Mixer's unary branch.